modules/custom/activity_basics/src/Plugin/ActivityContext/GroupContentInMyGroupActivityContext.php

```diff
@@ -109,7 +109,16 @@ public function getRecipients(array $data, int $last_id, int $limit): array {
       $group = $group_content->getGroup();
 
       $memberships = $group->getMembers($group->bundle() . '-group_manager');
-      $memberships = array_merge($memberships, $group->getMembers($group->bundle() . '-group_admin'));
+
+      // Retrieve all members with admin access.
```

Comment:
@volodymyr-sydor This check might have a huge performance impact. 
Also it seems redundant and not needed since in ticked we mentioned:
> It is important to understand that as a Group Administrator/Site Manager insider, the user is not a Group Manager. He should not receive the type of notification a group manager receives but only the same notification as a regular Group member.

Reply:
@ribel  done